src/utils/section-handler.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const fallback = require('mdast-util-to-hast/lib/handlers/root');
+const all = require('mdast-util-to-hast/lib/all');
+const wrap = require('mdast-util-to-hast/lib/wrap');
+
+/**
+ * Get the tag name for the specified section.
+ *
+ * @param {Node} section The MDAST section to get the tag name for
+ * @returns {string} The tag name for the section. Defaults to {@code div}.
+ */
+function getTageName(section) {
+  return (section.meta && section.meta.tagName) || 'div';
+}
+
+/**
+ * Get the class name for the specified section.
+ *
+ * @param {Node} section The MDAST section to get the class name for
+ * @returns {string} The class name for the section. Defaults to {@code hlx-section}.
+ */
+function getClass(section) {
+  return (section.meta && section.meta.className) || 'hlx-section';
```

Comment:
Same here, the YAML property should be `class`, even if it is slightly inconvenient to read it from JavaScript.

Also, what happens to the detected section types?